component/src/main/java/io/siddhi/extension/store/rdbms/RDBMSEventTable.java

```diff
@@ -1039,11 +1123,28 @@ private void batchProcessInsert(List<Object[]> addingRecords) throws ConnectionU
             //Filter update result index and adding to list.
             if (updateResultIndex[i] < 1) {
                 insertIndexList.add(lastUpdatedRecordIndex + i);
+            } else if (metrics != null) {
+                int count = updateResultIndex[i];
+                metrics.getUpdateCountMetric().inc(count);
+                metrics.getWritesCountMetrics().inc(count);
+                metrics.getTotalWriteMetrics().inc(count);
+                metrics.setRDBMSStatus(RDBMSStatus.PROCESSING);
             }
         }
         return insertIndexList;
     }
 
+    private int executedRowsCount(int[] updateResultIndex) {
+        int count = 0;
+        for (int resultIndex : updateResultIndex) {
+            if (resultIndex >= 1) {
+                count += resultIndex;
+            }
+        }
+        return count;
+    }
+
+
```

Comment:
Remove extra newline

Reply:
Resolve in d6916ba559d2adf41f0e9157bc043f8c05471185